libs/state/README.md

```diff
@@ -1,79 +1,70 @@
+#### [.select()](https://www.rx-angular.io/docs/state/api/rx-state#select)
 
-There are plenty of solutions available for managing these challenges on a **global level** (Akita, NgRx, NgXs, ...).
-None of them is dedicated to targeting the particular needs of the **component level**.
+Get a cached, distinct Observable of your state, with options to access single properties, transform single properties, or transform state slices.
+With reactive composition support from rxjs, this method gives you complete control over selected data.
 
-`@rx-angular/state` was specifically designed to give developers a tool for mastering **component state** without forcing
-them to use complex design patterns.
+#### [.get()](https://www.rx-angular.io/docs/state/api/rx-state#get)
 
-Its lightweight and intuitive API and the automatic subscription handling make `@rx-angular/state` the **perfect fit** for handling state in any Angular component.
+Retrieve your current state in an imperative manner.
+Whether you want the entire state or just a part of it, `.get()` makes it easy to access your data,
+with the ability to access deeply nested values.
 
-Using this library allows you to implement things like:
+#### [.set()](https://www.rx-angular.io/docs/state/api/rx-state#set)
 
-- merge global into local state
-- shared state selections
-- subscription-less interaction
-- hook into imperative functions (e.g. component lifecycle or HostBindings)
+Easily update one or many properties of your state. Whether you provide a partial state object or a function to calculate the new value,
+`.set()` provides two signatures for updating multiple properties or a single property by name.
 
-with very little effort in any component.
+#### [.hold()](https://www.rx-angular.io/docs/state/api/rx-state#hold)
 
-<p float="left">
-  <img src="https://raw.githubusercontent.com/rx-angular/rx-angular/main/libs/state/docs/images/state_API-names.png" width="49%" />
-  <img src="https://raw.githubusercontent.com/rx-angular/rx-angular/main/libs/state/docs/images/state_API-types.png" width="49%" />
-</p>
+Manage side-effects of your state with the `.hold()` method, which holds the trigger Observable and executes an optional handler function.
+With automatic subscription handling, this method is an effective way to manage side-effects without the hassle.
 
-## Key features
+#### [.setAccumulator()](https://www.rx-angular.io/docs/state/api/rx-state#setaccumulator)
 
-- Slim and intuitive API
-- Automated subscription handling
-- Intuitive way for handling ViewModels
-- Connect any Observable source to the state
-- Partial state updates
-- Reactive state selection
-- Lazy state (no BehaviorSubject)
-- Foundation for zone-less Angular applications
+Customize your state accumulation function with `.setAccumulator()`.
+Whether you want to update the accumulation logic for deep updates or solve immutability problems, this method is a powerful tool for fine-tuning your state management.
```

Comment:
Two things about this part. Why did u create custom texts for the landing page instead of just re-using what is stated right now in the docs as description text for each api?

I also do not find it so useful without any usage information at this stage. I anyway have to click on the link to see how it is used.